cos/models/Player.py

```diff
@@ -116,6 +116,8 @@ def get_turn_options(self):
         if self.turn_state == "main_turn":
             if self.can_buy_settlement():
                 turn_options.append("buy_settlement")
+            if self.can_trade_resources():
+                turn_options.append("trade_resource")
```

Comment:
I think there is a bug here in that this keeps appending the same resource over and over without cleaning itself up. In my test of the game I got multiple "ore" to show for a single player. Further, without cleanup after each turn, it results in someone being able to select a duplicate entry from a previous round.

Reply:
@davmmeyer fixed